src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -66,20 +66,19 @@ public function normalize($object, $format = null, array $context = [])
         foreach ($object->getResourceNameCollection() as $resourceClass) {
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
             $resourceShortName = $resourceMetadata->getShortName();
-            $definitions[$resourceShortName] = $this->getDefinitionSchema($resourceClass, $resourceMetadata);
 
             foreach ($resourceMetadata->getCollectionOperations() ?? [] as $operationName => $operation) {
```

Comment:
Both `foreach` loops are almost copy/pasted. Maybe this common block can be extracted to a `private` method?